packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-modal.tsx

```diff
@@ -15,11 +17,51 @@ const JurisdictionModal = observer(({ openPasswordModal }: TJurisdictionModalPro
 
     const { account_type, is_jurisdiction_modal_visible, toggleJurisdictionModal } = useCfdStore();
 
-    const modal_title = show_eu_related_content
-        ? localize('Choose a jurisdiction for your Deriv MT5 CFDs account')
-        : localize('Choose a jurisdiction for your Deriv MT5 {{account_type}} account', {
-              account_type: localize(getMT5Title(account_type.type)),
-          });
+    const [is_dynamic_leverage_visible, setIsDynamicLeverageVisible] = React.useState(false);
+
+    const toggleDynamicLeverage: React.MouseEventHandler<HTMLSpanElement> = event => {
+        event.stopPropagation();
+        setIsDynamicLeverageVisible(!is_dynamic_leverage_visible);
+    };
+
+    let modal_title;
+    if (is_dynamic_leverage_visible) {
+        modal_title = (
+            <div className='jurisdiction-modal__title'>
+                <span
+                    data-testid='back_icon'
+                    className='jurisdiction-modal__title-back'
+                    onClick={toggleDynamicLeverage}
+                >
+                    <Icon icon='IcArrowLeftBold' />
+                </span>
+                <Text weight='bold' color='prominent'>
+                    {localize('Get more out of Deriv MT5 Financial')}
+                </Text>
+            </div>
+        );
+    } else if (show_eu_related_content) {
+        modal_title = localize('Choose a jurisdiction for your Deriv MT5 CFDs account');
+    } else {
+        modal_title = localize('Choose a jurisdiction for your Deriv MT5 {{account_type}} account', {
+            account_type: localize(getMT5Title(account_type.type)),
+        });
+    }
```

Comment:
All of these should be in its own component
```suggestion
    let modal_title;
    if (is_dynamic_leverage_visible) {
        modal_title = (
            <div className='jurisdiction-modal__title'>
                <span
                    data-testid='back_icon'
                    className='jurisdiction-modal__title-back'
                    onClick={toggleDynamicLeverage}
                >
                    <Icon icon='IcArrowLeftBold' />
                </span>
                <Text weight='bold' color='prominent'>
                    {localize('Get more out of Deriv MT5 Financial')}
                </Text>
            </div>
        );
    } else if (show_eu_related_content) {
        modal_title = localize('Choose a jurisdiction for your Deriv MT5 CFDs account');
    } else {
        modal_title = localize('Choose a jurisdiction for your Deriv MT5 {{account_type}} account', {
            account_type: localize(getMT5Title(account_type.type)),
        });
    }
```

Reply:
Updated.